SSA/Projects/InstCombine/LLVM/Semantics.lean

```diff
@@ -34,6 +34,9 @@ def and {w : Nat} (x y : IntW w) : IntW w := do
 The ‘or’ instruction returns the bitwise logical inclusive or of its two
 operands.
 -/
+
+structure OrParams where
```

Comment:
why is this a `structure`? do you expect the params to increase? if it's only a `Bool` in LLVM then making this a `structure` sounds like unnecessary complexity